src/ring_theory/witt_vector/init_tail.lean

```diff
@@ -121,8 +121,12 @@ begin
   simp only [witt_polynomial_eq_sum_C_mul_X_pow, select_poly, alg_hom.map_sum, alg_hom.map_pow,
     alg_hom.map_mul, bind₁_X_right, bind₁_C_right, ← finset.sum_add_distrib, ← mul_add],
   apply finset.sum_congr rfl,
-  intros, congr' 2,
-  split_ifs; simp only [zero_pow (pow_pos hp.1.pos _), add_zero, zero_add],
+  refine λ m hm, mul_eq_mul_left_iff.mpr (or.inl _),
+  rw [ite_pow, ite_pow, zero_pow (pow_pos hp.out.pos _)],
```

Comment:
Can we replace the `[hp : fact p.prime]` at the top of this file with `[fact p.prime]`, get rid of the `include hp`, and replace this line with the following?

```lean
rw [ite_pow, ite_pow, zero_pow (pow_pos (fact.out (nat.prime p)).pos _)],
```